modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStoreTest.java

```diff
@@ -498,8 +498,7 @@ private static ByteBuffer copyOf(ByteBuffer buffer) {
     private static ByteBuffer createPageByteBuffer() {
         ByteBuffer buffer = ByteBuffer.allocateDirect(PAGE_SIZE).order(nativeOrder());
 
-        PageIo.setType(bufferAddress(buffer), MAX_IO_TYPE);
-        PageIo.setVersion(bufferAddress(buffer), 1);
+        new TestPageIo().initNewPage(GridUnsafe.bufferAddress(buffer), 0, PAGE_SIZE);
```

Comment:
I bet we already have a test io somewhere nearby, can we reuse it? Or they're all private?

Reply:
Reuse **org.apache.ignite.internal.pagememory.TestPageIoModule.TestPageIo**